client/src/components/DataFiles/DataFilesModals/DataFilesCopyModal.js

```diff
@@ -199,6 +204,14 @@ const DataFilesCopyModal = React.memo(() => {
           </div>
         </div>
       </ModalBody>
+      <div className="d-flex">
+        <ModalFooter>
+          <SectionMessage type="warning">
+            Files copied to Public Data will be avaliable to general public.{' '}
+            <b>This action cannot be reversed.</b>
+          </SectionMessage>
+        </ModalFooter>
+      </div>
```

Comment:
I believe we only want to show this message if the user has selected `Public Data` from the copy system selector dropdown.